ocw_import/api.py

```diff
@@ -93,11 +95,35 @@ def import_ocw2hugo_content(bucket, prefix, website):  # pylint:disable=too-many
                 log.exception("Error saving WebsiteContent for %s", s3_key)
 
 
-def convert_data_to_content(
-    filepath, data, website, course_home_uuid
+def convert_data_to_content(filepath, data, website, course_home_uuid):
```

Comment:
There are some conversion serializers available in `content_sync.serializers` : `ContentFileSerializerFactory`, `HugoMarkdownFileSerializer`, `JsonFileSerializer`, and `YamlFileSerializer`.  It would be nice to reuse them here instead of adding new functions, but they depend on a site config that is accurate, and currently it is not.  But something to keep in mind for the future, I will create a tech debt issue for it.

Reply:
On second thought, this whole import process should be going away eventually, so it's probably not worth creating an issue for it.

Reply:
Probably not, but we could still address it at a later date if we want